hack/verify-generate.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+if [[ -n $(git status --porcelain 2>/dev/null) ]]; then
```

Comment:
It may be worth  printing the outcome/output on error.

Reply:
good point... I've added it.
I hope that it will reveal why it's failing right now because I wasn't able to reproduce it locally so far.